hcloud/schema/storage_box.go

```diff
@@ -83,39 +83,46 @@ type StorageBoxCreateRequestAccessSettings struct {
 }
 
 // StorageBoxCreateResponse defines the schema of the response when
-// creating a storage box.
+// creating a Storage Box.
 type StorageBoxCreateResponse struct {
 	StorageBox StorageBox `json:"storage_box"`
 	Action     Action     `json:"action"`
 }
 
-// StorageBoxUpdateRequest defines the schema of the request to update a storage box.
+// StorageBoxUpdateRequest defines the schema of the request to update a Storage Box.
 type StorageBoxUpdateRequest struct {
-	Name   *string            `json:"name,omitempty"`
+	Name   string             `json:"name,omitempty"`
```

Comment:
This change feels backward. The fields in update requests will become optional for all three resources.